heltour/tournament/lichessapi.py

```diff
@@ -36,13 +36,18 @@ def _apicall(url, timeout=300, check_interval=0.1, post_data=None):
         if time_spent >= timeout:
             raise ApiWorkerError('Timeout for %s' % url)
 
+def _apicall_with_error_parsing(*args, **kwargs):
+    result = _apicall(*args, **kwargs)
+    if result == '':
+        raise ApiWorkerError('API failure')
+    if result.startswith("CLIENT-ERROR: "):
+        raise ApiClientError(f'API failure: {result}')
```

Comment:
There's one spot where ApiWorkerError is caught (tasks.py), can we fix/verify that to make sure it still catches the right exception?

Reply:
Will do

Reply:
I made ApiClientError inherit from the ApiWorkerError, which should allow any usages of except ApiWorkerError to continue to work.